grobid-core/src/main/java/org/grobid/core/jni/DeLFTModel.java

```diff
@@ -296,11 +305,11 @@ public void run() {
 
     private static String delft2grobidLabel(String label) {
         if (label.equals("O")) {
-            label = "<other>";
-        } else if (label.startsWith("B-")) {
-            label = label.replace("B-", "I-");
-        } else if (label.startsWith("I-")) {
-            label = label.replace("I-", "");
+            label = TaggingLabels.OTHER_LABEL;
+        } else if (label.startsWith(GenericTaggerUtils.START_ENTITY_LABEL_PREFIX_ALTERNATIVE)) {
+            label = label.replace(GenericTaggerUtils.START_ENTITY_LABEL_PREFIX_ALTERNATIVE, GenericTaggerUtils.START_ENTITY_LABEL_PREFIX);
+        } else if (label.startsWith(GenericTaggerUtils.START_ENTITY_LABEL_PREFIX)) {
+            label = label.replace(GenericTaggerUtils.START_ENTITY_LABEL_PREFIX, "");
```

Comment:
I would keep the original IOB strings and not use this `GenericTaggerUtils` stuff because it is unreadable. There is here something confusing about which annotation scheme is used, because actually it mixes here a Grobid prefix `START_ENTITY_LABEL_PREFIX` for testing a IOB prefix which is not the start of an entity but _inside_.

Reply:
As an alternative we could introduce clearly the annotation "scheme" which is used in the constant names, e.g. `TaggingLabels.OTHER_LABEL_IOB`, `TaggingLabels.OTHER_LABEL_GROBID`, `TaggingLabels.START_ENTITY_LABEL_PREFIX_IOB`, `TaggingLabels.START_ENTITY_LABEL_PREFIX_GROBID`, etc. but I am not sure it helps in term of readability.

Reply:
Indeed, this is not readable. 

I've pushed a change that should make more sense. Let me know what you think